plugins/modules/podman_container.py

```diff
@@ -595,6 +595,11 @@
       - If true, the first argument refers to an exploded container on the file
         system. The default is false.
     type: bool
+  secrets:
+    description:
+      - Add the named secrets into the container at /run/secrets.
+    type: list
+    elements: str
```

Comment:
not sure if `no_log` is needed here as well, but better to add

Reply:
ERROR: plugins/modules/podman_container.py:0:0: invalid-documentation: DOCUMENTATION.options.secrets.no_log: extra keys not allowed @ data['options']['secrets']['no_log']. Got True

Reply:
Ok, fair enough